web/src/components/predictions/leaderboard.js

```diff
@@ -0,0 +1,81 @@
+/** @jsx jsx */
+import React from 'react';
+import { jsx, Text, Box } from "theme-ui";
+import firebase from "gatsby-plugin-firebase";
+import Spacer from "../../components/core/spacer";
+import UpdateScore from "./update-score";
+
+// generate leaderboard
+const Leaderboard = ({user}) => {
+    const ref = firebase.database().ref("predictions/leaderboard");
+    let userIndex = 0;
+    let topScores;
+    ref.on('value',(snapshot) => {
+        const scores = snapshot.val();
+        const uids = Object.keys(scores).sort((a, b) => scores[b]["score"] - scores[a]["score"]);
+        let getIndex = {};
```

Comment:
This can be set to const I think
```suggestion
        const getIndex = {};
```